rholang/examples/tut-make-cell.rho

```diff
@@ -0,0 +1,37 @@
+new MakeCell in {
```

Comment:
losing here too:

```
*** formals mismatch:
 formals = [[channel unification_ptrn product]]
 actuals = [['Rholang-MakeCell--261298636 'wildcard 123 'Rholang-myGet-47677321 'Rholang-mySet--32894946]]
 in (FOp  produce), on (FO NameSpaceSBO 144453956)
```

rbl code:
```
(let [[MakeCell (generateFresh "MakeCell")]] (block ( (proc [] (run (compile (let [[[[Rholang819262138c50] [[Rholange097047ec0d8 Rholang267b530df618 Rholang974a4d24260e]]] (consume t [MakeCell] [**wildcard**] [['init 'get 'set]] #t)]] ( (proc [[init get set]] (let [[valueStore (generateFresh "valueStore")]] (block (block (produce t valueStore **wildcard** init) ( (proc [] (run (compile (let [[[[Rholangf433b954415a] [Rholang7b4a6cd05fdf]] (consume t [get] [**wildcard**] ['ack] #t)]] ( (proc [ack] (let [[[[Rholangb042e3c54d23] [Rholang2e4c67045015]] (consume t [valueStore] [**wildcard**] ['value] #f)]] ( (proc [[value]] (block (produce t valueStore **wildcard** value) (produce t ack **wildcard** value))) [Rholang2e4c67045015]))) Rholang7b4a6cd05fdf))))))) ( (proc [] (run (compile (let [[[[Rholang0e9a428569a8] [Rholangd905ffa9c4e8]] (consume t [set] [**wildcard**] ['pair] #t)]] ( (proc [pair] (let [[[[Rholang8f89b812e5d4] [Rholang8a20ce843cd0]] (consume t [valueStore] [**wildcard**] ['**wildcard**] #f)]] ( (proc [[**wildcard**]] (if (match? t pair [newValue ack]) ( (proc [[newValue ack]] (block (produce t valueStore **wildcard** newValue) (produce t ack **wildcard** #niv))) pair) #niv)) [Rholang8a20ce843cd0]))) Rholangd905ffa9c4e8))))))))) [Rholange097047ec0d8 Rholang267b530df618 Rholang974a4d24260e])))))) (let [[myGet (generateFresh "myGet")] [mySet (generateFresh "mySet")]] (block (produce t MakeCell **wildcard** 123 myGet mySet) (let [[ack (generateFresh "ack")]] (block (produce t myGet **wildcard** ack) (let [[[[Rholangd6a9594635c9] [Rholangb8367d3166be]] (consume t [ack] [**wildcard**] ['result] #f)]] ( (proc [[result]] (block (block ( display result) (produce t mySet **wildcard** [456 ack])) (let [[[[Rholang1a1a85a09229] [Rholang64b6cd529539]] (consume t [ack] [**wildcard**] ['**wildcard**] #f)]] ( (proc [[**wildcard**]] (block (produce t myGet **wildcard** ack) (let [[[[Rholang8a6d49cc356b] [Rholanga5ea389d9f46]] (consume t [ack] [**wildcard**] ['result] #f)]] ( (proc [[result]] ( display result)) [Rholanga5ea389d9f46])))) [Rholang64b6cd529539])))) [Rholangb8367d3166be]))))))))
```

Reply:
Now that I merged guardbotmk3:dev-fix-release-blockers, it runs, but a newline might be nice:

```
rosette> 123456[]
rosette> 
```